sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DescribeTableExec.scala

```diff
@@ -36,14 +36,33 @@ case class DescribeTableExec(
   override protected def run(): Seq[InternalRow] = {
     val rows = new ArrayBuffer[InternalRow]()
     addSchema(rows)
+    addPartitioning(rows)
```

Comment:
Since both hive and DescribeTableCommand (for v1 table) display partition information even extended not specified, so i put the addPartitioning before the if clause.
Correct me if this is not expected.